src/pywifes.py

```diff
@@ -2499,7 +2499,7 @@ def wifes_illumination(spatial_inimg,
         # need to fit this for each slitlet
     outfits[0].header.set('PYWIFES', __version__, 'PyWiFeS version')
     outfits.writeto(outimg, overwrite=True)
-    f1.close()
+    #f1.close()
```

Comment:
Better to delete this line and rename f2 to f1 throughout this function. Unless I've missed something...

Reply:
I deleted the commented line but I'll leave the variable as f2 for consistency because these f1, f2, and f3 variables are always defined in the same way in other functions in the scripts.

Reply:
Right, just saw that. Kinda means they're badly named across all functions, should be something like `spatial_fits = pyfits.open(spatial_fits_path)` (if I've understood it correctly.)

No problem, can change later if needed.